src/templates/item/components/details-damage.hbs

```diff
@@ -61,4 +61,37 @@
     {{/if}}
 </div>
 {{/if}}
+
+<div class="form-group-stacked">
+    <div class="header">
+        <label>{{localize "COSMERE.Item.Sheet.Damage.GrazeOverride"}}</label>
+        <div class="controls">
+            {{#if grazeInputCollapsed}}
+            <a data-action="toggle-graze-collapsed"
+                data-tooltip="COSMERE.Item.Sheet.Damage.GrazeShow"
+            >
+                <i class="fa-solid fa-chevron-left"></i>
+            </a>
+            {{else}}
+            <a data-action="toggle-graze-collapsed"
+                data-tooltip="COSMERE.Item.Sheet.Damage.GrazeHide"
+            >
+                <i class="fa-solid fa-chevron-down"></i>
+            </a>
+            {{/if}}
+        </div>
+    </div>
+    {{#if (not grazeInputCollapsed)}}
+    <div class="form-group formula">
+        <label>{{localize "COSMERE.Item.Sheet.Damage.GrazeFormula"}}</label>
+        <input name="system.damage.grazeOverrideFormula"
+            type="text"
+            value="{{item.system.damage.grazeOverrideFormula}}"
+            {{#if (not editable)}}
+            readonly
+            {{/if}}
+        >
+    </div>
+    {{/if}}
+</div>
```

Comment:
Graze is only relevant in the case of a skill test, right? I think this should be moved into the  `hasSkillTest` if-statement above.

Reply:
Yeah, now I think about it you're right. That's a little implicit in the way the rules describe it so didn't stand out to me late last night 😂
But I can't see an example where the official stats or special rules suggest this can be done outside of a roll. I think there are effects that remove the focus cost to do it? But not anything that allows you to apply graze damage without making an attack roll